ztag/schema.py

```diff
@@ -510,6 +508,15 @@ class CensysString(WhitespaceAnalyzedString):
     "metadata": local_metadata,
 })
 
+# The oracle ztag transform is a plain copy of the "handshake" field.
+ztag_oracle = zgrab2_oracle.oracle_scan_response["result"]["handshake"]
+
+ztag_mssql = SubRecord({
```

Comment:
If we want to enforce validation of these new protocols at the ESLoader level, we need to add an explicit `validation_policy="error"` to them in the schema. This could be out of scope for this PR, but I want to be sure we're on the same page.